src/api/app/views/webui2/webui/staging/workflows/_problems.html.haml

```diff
@@ -2,20 +2,31 @@
   max_problems = 5
   project_class = staging_project.name.tr(':', '_')
   item_project_class = 'hidden-item-' + project_class
+  left_problems = max_problems
```

Comment:
what `left_problems` means?

I don't think this variable is needed...

Reply:
@ana06 i will check if i can do it without, otherwise i will make the name a bit more meaningful :)

Reply:
@Ana06 check again, i got rid of the variable